xla/stream_executor/gpu/gpu_command_buffer_test.cc

```diff
@@ -1,1705 +0,0 @@
-/* Copyright 2023 The OpenXLA Authors.
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-==============================================================================*/
-
-#include "xla/stream_executor/gpu/gpu_command_buffer.h"
```

Comment:
Why it's deleted? I don't think we have any other tests for low level se::CommandBuffer APIs

Reply:
I think most of the command functions are moved to command_buffer_cmd now,  and GpuCommandBuffer class is deleted. CommandBuffer now only has graph node create and update API, graph executor instantiate and execute APIs, which can all be covered by command_buffer_cmd_test and command_buffer_thunk_test now.

Reply:
`command_buffer_cmd_test` is an xla level test with bunch of other details, we need a test for low level implementation details, and this was the one. It has a lot of duplication, but it allows to test implementation details without testing bunch of other code